bin/docker_lib.sh

```diff
@@ -5,7 +5,7 @@ if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
 fi
 
 # standard checks
-set -ex
```

Comment:
Now doesn't seem like the right time to be killing all the logging that happens in a build.

Reply:
Mandar, removing logging could potentially make the build less reproducible.